bpmsuite/bpmsuite70-businesscentral-monitoring-with-smartrouter.json

```diff
@@ -57,22 +57,7 @@
             "displayName": "KIE Admin Password",
             "description": "KIE administrator password",
             "name": "KIE_ADMIN_PWD",
-            "from": "[a-zA-Z]{6}[0-9]{1}!",
-            "generate": "expression",
-            "required": false
-        },
-        {
-            "displayName": "KIE Server Controller User",
-            "description": "KIE server controller username (adds the user)",
-            "name": "KIE_SERVER_CONTROLLER_USER",
-            "value": "controllerUser",
-            "required": false
-        },
-        {
-            "displayName": "KIE Server Controller Password",
-            "description": "KIE server controller password (adds the user)",
-            "name": "KIE_SERVER_CONTROLLER_PWD",
-            "value": "controllerPassword",
+            "value": "adminPassword",
```

Comment:
This is set in order to match the user/pass set on the exec. server credentials to access the maven repo